source/renderer/app/components/wallet/WalletSendForm.tsx

```diff
@@ -106,9 +112,8 @@ class WalletSendForm extends Component<Props, State> {
   static contextTypes = {
     intl: intlShape.isRequired,
   };
-  // @ts-ignore ts-migrate(2416) FIXME: Property 'state' in type 'WalletSendForm' is not a... Remove this comment to see the full error message
   state = {
-    formFields: {},
+    formFields: {} as State['formFields'],
```

Comment:
Instead of casting here you could tweak the type in two ways to
```
type State = {
  formFields: {} || {  // either empty object or object always having all of the properties
    receiver: Field;
    adaAmount: Field;
    assetFields: Record<string, Field>;
    assetsDropdown: Record<string, Field>;
  }
```
or make all properties optional:
```
type State = {
  formFields: {
    receiver?: Field;
    adaAmount?: Field;
    assetFields?: Record<string, Field>;
    assetsDropdown?: Record<string, Field>;
  }
```